### PR TITLE
Wappalyzer: Upgrade

### PR DESCRIPTION
**Pull request by kingthorin**
Align with AliasIO/Wappalyzer v5.7.4.
* Refresh apps.json (removed two failing patterns).
* Refresh image collection.
* Update CHANGELOG.md.

For future reference, while execution of `ExtensionWappalyzer.main` may
display some parsing exceptions the extension does actually handle
things ok. However, those patterns should be removed from apps.json not
prevent ZAP from erroring on startup. (Future plan: remove main method,
and replace with a UnitTest.)

### CODE REVIEW

File: addOns/wappalyzer/CHANGELOG.md [hunk 1]
```diff
@@ -6,7 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ## Unreleased
 
 - Switch to using re2j where possible - results in significant performance improvements.
-- Added version information column to Wappalyzer Results
+- Added version information column to Wappalyzer Results.
+- Updated to align with AliasIO/WAppalyzer release v5.7.4.
```
Comment (thc202) [inline addOns/wappalyzer/CHANGELOG.md]: There's a typo in Wappalyzer.

File: addOns/wappalyzer/src/main/resources/org/zaproxy/zap/extension/wappalyzer/resources/icons/Apple.svg [hunk 2]
```diff
@@ -1,23 +1,23 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (thc202) [inline addOns/wappalyzer/src/main/resources/org/zaproxy/zap/extension/wappalyzer/resources/icons/Apple.svg]: Line ending changes? (The other new SVG files could also use \n.)

File: addOns/wappalyzer/CHANGELOG.md (hunk 1 above)
Comment (kingthorin) [reply addOns/wappalyzer/CHANGELOG.md]: Oops, fix coming.

File: addOns/wappalyzer/src/main/resources/org/zaproxy/zap/extension/wappalyzer/resources/icons/Apple.svg (hunk 2 above)
Comment (kingthorin) [reply addOns/wappalyzer/src/main/resources/org/zaproxy/zap/extension/wappalyzer/resources/icons/Apple.svg]: Hmm I’ll check

Comment (kingthorin) [reply addOns/wappalyzer/src/main/resources/org/zaproxy/zap/extension/wappalyzer/resources/icons/Apple.svg]: I've checked about a half dozen, I can fix them manually (outside of eclipse). But of those that I checked they were actually CRLF and are now LF.

Let me know what I should do.

Comment (kingthorin) [reply addOns/wappalyzer/src/main/resources/org/zaproxy/zap/extension/wappalyzer/resources/icons/Apple.svg]: Ok I think I got the "changes" related to EOL sorted. (For this PR)

**Comment by thc202** [comment]
Thank you!

**Pull request merged**